### ISSUE TITLE
Updating the task description via updateTask, updates the title instead.

### ISSUE THREAD

**Issue by Einzelganger** [fguillot/kanboard#2167]
### Expected behaviour
I expected the description of the given task to include some text. 

### Actual behaviour
The task's title gets modified.

### Steps to reproduce

1. create a connection
2. $client->updateTask($id = $taskID, $description = $newtext);

### Configuration

Copy and paste the configuration section from the Kanboard settings page or fill these fields:

- Kanboard version: 1.0.27
- Database type and version: 
- PHP version: 5.6
- OS: Linux
- Browser: Firefox

**Comment by jtourt** [comment]
The variables that you are creating are not seen by the method.  As a learning exercise, change your code to the following and see if it runs the same:
$client->updateTask($banana = $taskID, $pie = $newtext);

Study the client example with named arguments on this page: https://github.com/fguillot/JsonRPC
$client->updateTask(['id' => $taskID, 'description' => $newtext]);

You have been using "positional parameters," which as the name suggests relate to the positional order that the arguments are specified.  Note the order of parameters in the documentation for updateTask.  http://kanboard.net/documentation/api-task-procedures

updateTask
•Purpose: Update a task
•Parameters: 
◦id (integer, required)
◦title (string, optional)
◦color_id (string, optional)
◦owner_id (integer, optional)
◦date_due: ISO8601 format (string, optional)
◦description Markdown content (string, optional)

id is the first parameter.
title is the second parameter.
color_id is the third parameter.
owner_id is the fourth parameter.
date_due is the fifth parameter.
description is the sixth parameter.

You are passing two arguments.  The first goes into id.  The second goes into title.

TL;DR: Use named arguments or specify arguments in the right order.

**Comment by Einzelganger** [comment]
Thank you very much for the clarification and the example. I am new to PHP and did not realize this. 
Since the problem is solved, clossing issue.
